tests/misc/default_board_config/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.13.1)
+include($ENV{ZEPHYR_BASE}/cmake/app/boilerplate.cmake NO_POLICY_SCOPE)
+project(default_config)
+
+FILE(GLOB app_sources src/*.c)
```

Comment:
Maybe `main.c` could just be listed directly here, to avoid accidentally globbing up non-checked-in `.c` files.